src/workflows/bidsChangeSuffix.m

```diff
@@ -0,0 +1,139 @@
+function bidsChangeSuffix(varargin)
+  %
+  % USAGE::
+  %
+  %   bidsChangeSuffix(opt, newSuffix, 'filter', struct([]), 'force', false)
+  %
+  % :param opt: BIDS input dataset must be defined in ``opt.dir.input``;
+  % To test the output, set ``opt.dryRun`` to ``true``.
+  % :type opt: structure
+  %
+  % :param newSuffix: TODO: add checks on newSuffix to make sure it only contains [a-zA-Z0-9]
+  % :type newSuffix: string
+  %
+  % :param filter: structure to decide which files to include. Default:
+  % ``struct([])`` for no filter
+  % :type filter: structure
+  %
+  % :param force: If set to ``true`` it will overwrite already existing files. Default: ``false``
+  % :type force: boolean
+  %
+  %
+  % EXAMPLE::
+  %
+  %     opt.dir.input = path_to_dataset;
```

Comment:
is `input` the raw or the derivatives folder? 

can it be defined either with `opt.dir.raw` `opt.dir.derivatives` for path variables consistency?